data/presets/shop/newsagent.json

```diff
@@ -7,5 +7,8 @@
     "tags": {
         "shop": "newsagent"
     },
-    "name": "Newspaper/Magazine Shop"
+    "name": "Newspaper/Magazine Shop",
+    "aliases": [
+        "Newsstand"
```

Comment:
This is a much less clunky name than “Newspaper/Magazine Shop”. Or are we concerned about overlap with `shop=kiosk`?

Reply:
Well there is real-world overlap, but I am not concerned. I'll add newsstand to the list of terms for kiosk, so when searching for newsstand, one gets first this preset but the kiosk preset will probably be shown below